### PR TITLE
Support to add Kibana annotations at deployment level

### PR DESCRIPTION
**Pull request by framsouza**
This PR adds support to insert Kibana annotations at Deployment level

### CODE REVIEW

File: metricbeat/requirements.lock [hunk 1]
```diff
@@ -2,5 +2,5 @@ dependencies:
 - name: kube-state-metrics
   repository: https://charts.helm.sh/stable
   version: 2.4.1
-digest: sha256:948dca129bc7c16b138ed8bcbdf666c324d812e43af59d475b8bb74a53e99778
-generated: "2020-10-30T18:58:57.381827+01:00"
+digest: sha256:2d4e8265cb60d5c0f613217b35ab18846969c86a1a9fdf77cfa75c291f18934a
```
Comment (jmlrt) [inline metricbeat/requirements.lock:5]: This change isn't related

**Review by jmlrt — CHANGES_REQUESTED** [review]

File: metricbeat/requirements.lock (hunk 1 above)
Comment (framsouza) [reply metricbeat/requirements.lock]: @jmlrt I just put back the original requirement.lock and push it again

**Review by jmlrt — CHANGES_REQUESTED** [review]
Can you add the value description in the README and a python test?

**Comment by framsouza** [comment]
@jmlrt done

**Review by jmlrt — APPROVED** [review]
LGTM⛴

**Pull request merged**